### DISCUSSION TITLE
Issue With  one degree of freedom simulation.

### DISCUSSION THREAD

**Discussion by aravindvenkatachalapathy** [NatLabRockies/ROSCO#271] [General]
When Running the One DOF Simulation I'm getting the followinf error. What might be the possible problem here?
![image](https://github.com/NREL/ROSCO/assets/43172847/369390e0-e33e-49bf-8718-606d8caea0d4)

**Comment by dzalkind** [comment]
I don't see anything that could cause an error on line 58 of `ReadSetParameters.f90`.  What do you see there? Are you using the latest version of ROSCO?

What have you changed from the working example?

**Reply by aravindvenkatachalapathy** [reply]
No, I haven't changed anything from the working example, and I'm using the latest version of ROSCO. And for line 58 in ReadSetParameters.f90 I have attached the screenshot.
![image](https://github.com/NREL/ROSCO/assets/43172847/528d5516-c8ee-48bd-82d3-0d6e335525be)

**Reply by dzalkind** [reply]
Do other examples work with that ROSCO dynamic library?  Have you tried a pre-compiled version and then updating `lib_name` in the example script?
